js/hitbtc3.js

```diff
@@ -530,9 +530,9 @@ module.exports = class hitbtc3 extends Exchange {
             let withdrawEnabled = undefined;
             for (let j = 0; j < rawNetworks.length; j++) {
                 const rawNetwork = rawNetworks[j];
-                let networkId = this.safeString (rawNetwork, 'protocol', '');
-                if (networkId.length === 0) {
-                    networkId = this.safeString (rawNetwork, 'network', '');
+                let networkId = this.safeString (rawNetwork, 'protocol');
+                if (networkId !== undefined) {
+                    networkId = this.safeString (rawNetwork, 'network');
```

Comment:
This should be `=== undefined`, but it should just be `safeString2` I'll update it